no member named 'ForceSet' in 'v8::Object'

Hi! I've just upgraded to node 10 and now I cannot rebuild opencv4nodejs. I got the following error:

In file included from ../../nan/nan.h:192:
../../nan/nan_maybe_43_inl.h:112:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);

Thanks!

--- a/lib/commons.js
+++ b/lib/commons.js
@@ -11,7 +11,11 @@ function resolvePath(filePath, file) {
 const defaultDir = '/usr/local';
 
 function getLibDir() {
-  return resolvePath(process.env.OPENCV_LIB_DIR) || `${defaultDir}/lib`;
+  const libPath = resolvePath(process.env.OPENCV_LIB_DIR)
+  if (process.platform === 'win32' && !libPath) {
+    throw new Error('OPENCV_LIB_DIR is not defined')
+  }
+  return libPath || `${defaultDir}/lib`;
 }
 
 module.exports = {

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
   "gypfile": true,
   "dependencies": {
     "macro-inferno": "^0.2.1",
-    "nan": "^2.7.0",
+    "nan": "^2.10.0",
     "native-node-utils": "0.1.2",
     "opencv-build": "^0.0.12"
   },

